<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_geruniforme</class>
 <widget class="QWidget" name="Form_geruniforme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>622</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>622</width>
    <height>500</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Gerenciamento de uniformes</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_busca">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_busca">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>üîç</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Funcion√°rio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_registrar">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Registra a movimenta√ß√£o atual</string>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>601</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Entrega</string>
    </attribute>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Observa√ß√£o</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_obs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>551</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_qtd">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>190</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Qtd.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_tamanho">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>P</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>M</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>XG</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamanho</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_descricaouniforme">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Camisa ML</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cal√ßa</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Uniforme</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_adicionar">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Adiciona uniforme √† movimenta√ß√£o atual</string>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_remover">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Remove item selecionado</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>A√ß√£o</string>
     </property>
     <widget class="QRadioButton" name="radioButton_retirada">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Retirada</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_devolucao">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Devolu√ß√£o</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget_movimentacao">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>161</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>A√ß√£o</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descri√ß√£o</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tamanho</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Observa√ß√£o</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Hist√≥rico</string>
    </attribute>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Hist√≥rico:</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_movimentacao_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>161</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::DotLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>A√ß√£o</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Descri√ß√£o</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tamanho</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Observa√ß√£o</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_idfuncionario">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="digitCount">
    <number>5</number>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_busca</tabstop>
  <tabstop>pushButton_busca</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioButton_retirada</tabstop>
  <tabstop>radioButton_devolucao</tabstop>
  <tabstop>comboBox_descricaouniforme</tabstop>
  <tabstop>comboBox_tamanho</tabstop>
  <tabstop>spinBox_qtd</tabstop>
  <tabstop>textEdit_obs</tabstop>
  <tabstop>pushButton_adicionar</tabstop>
  <tabstop>pushButton_remover</tabstop>
  <tabstop>pushButton_registrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
